block_production.py is flaky

Fails with

```
Traceback (most recent call last):
  File "tests/sanity/block_production.py", line 76, in <module>
    assert min_common() + 2 >= height, heights_report()
AssertionError: None
```

http://nayduck.eastus.cloudapp.azure.com:3000/#/test/7836

--- a/pytest/tests/sanity/block_production.py
+++ b/pytest/tests/sanity/block_production.py
@@ -31,6 +31,14 @@ last_common = [[0 for _ in nodes] for _ in nodes]
 
 height_to_hash = {}
 
+# the test relies on us being able to query heights faster
+# than the blocks are produced. Validating store takes up
+# to 350ms on slower hardware for this test, multiplied
+# by four nodes querying heights every 1 second becomes
+# unfeasible
+for node in nodes:
+    node.stop_checking_store()
+
 
 def min_common():
     return min([min(x) for x in last_common])

